runtime/Python2/src/antlr4/TokenStreamRewriter.py

```diff
@@ -104,7 +105,7 @@ def getDefaultText(self):
 
     def getText(self, program_name, start, stop):
         """
-        :type interval: Interval.Interval
+        :type interval: IntervalSet.Interval
```

Comment:
No longer valid

Reply:
OK, I'll fix it in another PR with some other fixes.